### ISSUE TITLE
Dakar 18 (767390)

### ISSUE THREAD

**Issue by leillo1975** [ValveSoftware/Proton#2113]
# Compatibility Report
- Name of the game with compatibility issues: Dakar 18
- Steam AppID of the game: 767390

## System Information
- GPU: Nvidia Geforce GTX-1050Ti
- Driver/LLVM version: nvidia 415.23
- Kernel version: 4.18
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/leillo1975/e1391f70160a93280e48a5d20fba5369
- Proton version: 3.16-5 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-767390.log](https://github.com/ValveSoftware/Proton/files/2700608/steam-767390.log)

## Symptoms 
The game works almost perfect, with some not serious stutterers, but it don't recognize my Logitech G27 Wheel. I try to make the controls assistant, but is impossible that the game detects my wheel. 
If I use my Steam Controller, I don't have any problem.
About performance, in medium settings, the FPS are between 50-70 with my GTX-1050Ti

## Reproduction

I only connects my G27 wheel before starts the game, and then try to detect it (impossible)

**Comment by leillo1975** [comment]
This Christmas I received a G29 as gift  (thank you Santa) and it has the same problem. It was something to be expected, but I leave it here in case it can useful to someone.

**Comment by leillo1975** [comment]
"Desafio Ruta 40" and "Desafío Inca" free DLC's, works also with Proton. I have a Issue, but I think that this is a game problem: Voices and Subtitles in this DLC's are showed in English, and in my settings are configured in Spanish.

**Comment by leillo1975** [comment]
> "Desafio Ruta 40" and "Desafío Inca" free DLC's, works also with Proton. I have a Issue, but I think that this is a game problem: Voices and Subtitles in this DLC's are showed in English, and in my settings are configured in Spanish.

Is a game problem, not Proton Problem. Confirmed

**Comment by leillo1975** [comment]
After today game update, now my G29 wheel is recognized but is not possible to assign axis in configuration. When the game starts, the wheel autocenters and gets hard to turn. When I go to devices I can see my wheel, but then is impossible to configure it:
https://steamuserimages-a.akamaihd.net/ugc/996891527335493631/104B22D5B9ED43D2C3D54C38CFC1C944F2DA77E3/

I have also noticed that like other Unreal Engine 4 titles ( https://github.com/ValveSoftware/Proton/issues/2430 ), videos are not shown. For example, the final podium video is not shown.

**Comment by kisak-valve** [comment]
Looks like this game would benefit from Media Foundation support.

**Comment by nsivov** [comment]
Right, it's loading mfplay.dll, similar to issue #1320 in that sense.

**Comment by leillo1975** [comment]
Logitech G29 continues not working with Proton 4.2. The wheel is detected correctly, but axis and buttons are not recognized when you move or press it.

**Comment by leillo1975** [comment]
With the last version of Proton (4.11-10),The game continues without recognizing my steering wheel. Is the only racing game that I own that don't work with my G29. With my Steam controller I don't have any problem. Can you take a look to this problem?

NOTE: I found that when I change my G29 to G27 or DFGT compatibility mode, the wheel is recognized as G29, but don't detect the direction and is impossible to reasign the keys

**Comment by leillo1975** [comment]
Have this game the same problem as [WRC 7](https://github.com/ValveSoftware/Proton/issues/1183#issuecomment-638259624)?

**Comment by mrpippy** [comment]
I just tried this out with 5.0-8 RC and my G25, it detects the wheel but can't learn the axes correctly. Brake pedal, shift paddles, and other buttons aren't detected correctly by the assistant, and the manual configuration doesn't seem to have a way to override those.
The game directory has JSON-formatted configurations (although not one for the G25), editing these may be needed.

**Comment by leillo1975** [comment]
With Proton 5.13-1 this problem continues. No changes on Wheel detection

**Comment by leillo1975** [comment]
I can say that with the last version of Proton, Dakar 18 works with my G29 Logitech wheel, with Force Feedback!!! You only have to follow the device assistant to change the pedals axis
